src/components/molecules/ProfileMenu/ProfileMenu.vue

```diff
@@ -44,14 +63,18 @@
 
 <script setup lang="ts">
 import ROUTES from '@/router/domain-routes'
+import { ref } from 'vue'
 
 interface ProfileMenuProps {
   username: string
   avatar: string
+  selected: string
```

Comment:
mấy cái item của Profile Menu sẽ tự active lên khi truy cập vào url tương ứng chứ ko phải truyền props vào
em fix lại cái component Profile Menu luôn nhé
https://router.vuejs.org/guide/essentials/active-links#Configuring-the-classes